Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -24,9 +24,9 @@ $connection = Get-VssConnection -TaskContext $distributedTaskContext
 
 $resources = Get-EnvironmentResources -EnvironmentName $environmentName -ResourceFilter $machineNames -Connection $connection -ErrorAction Stop
 
-$machineUserName = Get-EnvironmentProperty -EnvironmentName $environmentName -Key "Username" -Connection $connection -ErrorAction Stop
+$machineUserName = Get-EnvironmentProperty -EnvironmentName $environmentName -Key "Microsoft-Vslabs-MG-Resource-Username" -Connection $connection -ErrorAction Stop
```

Comment:
this work I have done and validating , but to get unblock we need to push this changes .